Feat/auto update (#1311)
NordicSemiconductor/pc-nrfconnect-launcher: Changelog.md

## Unreleased

### Added

> - Auto-update of apps which enable this when opening the app.

Reviewer: ```suggestion
- Updating of apps when clicking **Open** (for apps that enable this flow).
```
